<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Заполнение депо
 (Внимание! При полном заполнении депо произойдёт
сброс жидкости в опреснительную систему)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="depot">
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="work_label">
      <property name="text">
       <string>Качаем морскую воду в накопительное депо, под музыку!</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Режимы насоса</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="main_mode">
         <property name="text">
          <string>Основной режим</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="idle_mode">
         <property name="text">
          <string>Холостой режим</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pump">
      <property name="text">
       <string>Активировать рабочий цикл насоса</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="memo1">
      <property name="text">
       <string>Памятка: прежде чем начинать работу,
убедитесь что включён основной режим!</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="treck_label">
         <property name="text">
          <string>Список треков</string>
         </property>
         <property name="buddy">
          <cstring>treck_list</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="treck_list"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="now_tr_label">
         <property name="text">
          <string>Сейчас звучит трек:</string>
         </property>
         <property name="buddy">
          <cstring>playing_treck</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="playing_treck"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="memo2">
      <property name="text">
       <string>   Выбирайте трек согласно настроению и текущей задаче.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_17">
    <property name="title">
     <string>Опреснительная установка №17.  Меню управления насосом.</string>
    </property>
   </widget>
   <addaction name="menu_17"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
